appinventor/components/src/com/google/appinventor/components/runtime/File.java

```diff
@@ -317,6 +324,16 @@ public void GotText(String text) {
     // invoke the application's "GotText" event handler.
     EventDispatcher.dispatchEvent(this, "GotText", text);
   }
+  /**
+   * Event indicating that a request has finished.
+   *
+   * @param text write to the file
+   */
+  @SimpleEvent (description = "Event indicating that the contents of the file have been written.")
+  public void AfterText(String text) {
```

Comment:
It would be better to name this event AfterFileSaved to match the SaveFile name for the method.